advocacy_docs/pg_extensions/wait_states/installing.mdx

```diff
@@ -3,12 +3,41 @@ title: Installing EDB Wait States
 navTitle: Installing
 ---
 
-## On Linux 
+## On all Linux systems except Debian and Ubuntu
 
-To install EDB Wait States on a Linux system:
+To install EDB Wait States on any Linux operating system except Debian or Ubuntu:
 
-1.  Install EDB Wait States with the `edb-asxx-server-edb_wait_states` RPM package, where `xx` is the EDB Postgres Advanced Server version number.
+1.  Enter the following command:
 
+  ```shell
+  sudo <package-manager> -y install <install-command>
+  ```
+
+  Where: 
+  
+    - `<package-manager>`is the package manager used with your operating system:
+
+      | Package manager | Operating system     |
+      | --------------- | -------------------------------- |
+      | dnf             | RHEL 8/9 and derivatives         |
+      | yum             | RHEL 7 and derivatives, CentOS 7 |
```

Comment:
Wouldn't CentOS 7 be a derivative of RHEL 7?

Reply:
@martinmarques Technically, it probably is but this is the language we use in many places to avoid confustion